enterprise/models.py

```diff
@@ -397,13 +398,23 @@ def get_course_enrollment_url(self, course_key):
         Returns:
             (str): Enterprise landing page url.
         """
-        url = urljoin(
-            get_configuration_value('LMS_ROOT_URL', settings.LMS_ROOT_URL),
-            reverse(
-                'enterprise_course_enrollment_page',
-                kwargs={'enterprise_uuid': self.uuid, 'course_key': course_key}
+        if self.enable_learner_portal:
+            url = '{}/{}/course/{}'.format(
+                get_configuration_value(
+                    'ENTERPRISE_LEARNER_PORTAL_BASE_URL',
+                    settings.ENTERPRISE_LEARNER_PORTAL_BASE_URL
+                ),
+                self.slug,
+                course_key,
+            )
```

Comment:
I wonder if it would make sense to move these lines into a util function or otherwise since it's used a few times, both in the tests and the actual code? That way, `'{}/{}/course/{}` is only defined once.